### PR TITLE
Added a broadcasted message when a vehicle pool is activated.

### PR DESCRIPTION
**Pull request by LordDz**
Vehicle pools are really important to the team, but they're not broadcast when constructed.
As a logi player I'd like this to be broadcast so I don't have to spend time on telling the team this.

### CODE REVIEW

File: DH_Engine/Classes/DHPlatoonVPMessage.uc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+//==============================================================================
+// Darkest Hour: Europe '44-'45
+// Darklight Games (c) 2008-2023
+//==============================================================================
+
+class DHPlatoonVPMessage extends ROGameMessage
```
Comment (cmbasnett) [inline DH_Engine/Classes/DHPlatoonVPMessage.uc:6]: I would just rename this to `DHVehiclePoolMessage`.

File: DH_Engine/Classes/DHPlatoonVPMessage.uc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    abstract;
+
+    var localized string ActivatedMessage;
+    var localized string OverrunMessage;
+    var localized string DestroyedMessage;
+    var localized string ConstructedMessage;
```
Comment (cmbasnett) [inline DH_Engine/Classes/DHPlatoonVPMessage.uc:12]: Indentation is incorrect here, just bring it to the left one.

File: DH_Engine/Classes/DHPlatoonVPMessage.uc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    optional PlayerReplicationInfo RelatedPRI_2,
+    optional Object OptionalObject
+    )
+{
+    switch (Switch)
+    {
+        case 0:
+            return default.ActivatedMessage;
+        case 2:
+            return default.OverrunMessage;
```
[the remark this answers is not in the document]
Comment (LordDz) [reply DH_Engine/Classes/DHPlatoonVPMessage.uc]: @cmbasnett I've removed the unused messages and fixed the indent.

File: DH_Engine/Classes/DHVehiclePoolMessage.uc [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    )
+{
+    switch (Switch)
+    {
+        case 0:
+            return default.ActivatedMessage;
+        case 3:
+            return default.DestroyedMessage;
+        case 4:
+            return default.ConstructedMessage;
```
Comment (cmbasnett) [inline DH_Engine/Classes/DHVehiclePoolMessage.uc:27]: This one is also unused it seems.

**Review by cmbasnett — CHANGES_REQUESTED** [review]

File: DH_Engine/Classes/DHVehiclePoolMessage.uc (hunk 4 above)
Comment (LordDz) [reply DH_Engine/Classes/DHVehiclePoolMessage.uc]: Fixed!

**Review by cmbasnett — APPROVED** [review]

**Pull request merged**